code_generator/src/Generation/Generators/CommonGeneratorUtils.cs

```diff
@@ -6,5 +6,10 @@ public static string GetGeneratedHeader()
         {
             return "// ===========\r\n// DO NOT EDIT - this file is automatically regenerated.\r\n// ===========";
         }
+
+        public static string IndentString(string input)
```

Comment:
This doesn't just indent a string, could this have a better name?

Reply:
Yes probably, `IndentEveryNewline` ? 😬